### PR TITLE
add button and title to homepage

### PR DESCRIPTION
**Pull request by sarah-ogrady**
@Annedj can you please review this. What do you think of the button?

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,3 +1,5 @@
-<% if user_signed_in? %>
-  <%= current_user.email %>
-<% end %>
+<div class="container">
+  <h1>Amazing app</h1>
+  <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Architecto consequuntur vel illo atque id ipsa! Voluptatum sint accusantium, quidem, iste magnam doloribus illum inventore dolore quisquam, dolorum expedita consectetur quod.</p>
+  <%= link_to 'create', '#', class: 'btn btn-success'  %>
```
Comment (Annedj) [inline app/views/pages/home.html.erb:4]: Really like this green! 👌

**Review by Annedj — APPROVED** [review]
Looks good to me! Great home page 🙌

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (sarah-ogrady) [reply app/views/pages/home.html.erb:4]: Please can you merge if you are happy?

Comment (sarah-ogrady) [reply app/views/pages/home.html.erb:4]: @Annedj

**Pull request merged**